externs/olx.js

```diff
@@ -7443,6 +7444,16 @@ olx.tilegrid.TileGridOptions.prototype.extent;
 
 
 /**
+ * Zoom levels to load from a tile source. This array defines a subset of the
+ * available zoom levels. When configured, only tiles for the specified zoom
+ * levels will be requested from a tile source.
+ * @type {Array.<number>|undefined}
+ * @api
+ */
+olx.tilegrid.TileGridOptions.prototype.loadZooms;
```

Comment:
Not sure. I tend to like `levels` better indeed, but it might make users think it's a replacement for `resolutions`.